core-services/image-mirroring/openshift/mapping_origin_4_9

```diff
@@ -14,6 +14,9 @@ registry.ci.openshift.org/origin/4.9:aws-cloud-controller-manager quay.io/opensh
 registry.ci.openshift.org/origin/4.9:aws-ebs-csi-driver quay.io/openshift/origin-aws-ebs-csi-driver:4.9 quay.io/openshift/origin-aws-ebs-csi-driver:4.9.0 quay.io/openshift/origin-aws-ebs-csi-driver:latest
 registry.ci.openshift.org/origin/4.9:aws-ebs-csi-driver-operator quay.io/openshift/origin-aws-ebs-csi-driver-operator:4.9 quay.io/openshift/origin-aws-ebs-csi-driver-operator:4.9.0 quay.io/openshift/origin-aws-ebs-csi-driver-operator:latest
 registry.ci.openshift.org/origin/4.9:aws-ebs-csi-driver-operator-test quay.io/openshift/origin-aws-ebs-csi-driver-operator-test:4.9 quay.io/openshift/origin-aws-ebs-csi-driver-operator-test:4.9.0 quay.io/openshift/origin-aws-ebs-csi-driver-operator-test:latest
+registry.ci.openshift.org/origin/4.9:aws-efs-csi-driver quay.io/openshift/origin-aws-efs-csi-driver:4.9 quay.io/openshift/origin-aws-efs-csi-driver:4.9.0 quay.io/openshift/origin-aws-efs-csi-driver:latest
+registry.ci.openshift.org/origin/4.9:aws-efs-csi-driver-operator quay.io/openshift/origin-aws-efs-csi-driver-operator:4.9 quay.io/openshift/origin-aws-efs-csi-driver-operator:4.9.0 quay.io/openshift/origin-aws-efs-csi-driver-operator:latest
+registry.ci.openshift.org/origin/4.9:aws-efs-utils-base quay.io/openshift/origin-aws-efs-utils-base:4.9 quay.io/openshift/origin-aws-efs-utils-base:4.9.0 quay.io/openshift/origin-aws-efs-utils-base:latest
```

Comment:
I think we don't need to mirror this image, it's used only as a base image for aws-efs-csi-driver and it's useless otherwise.

Reply:
(not sure if the automation needs to publish all CI images on quay)

Reply:
Removed